courts_db/data/courts.json

```diff
@@ -18890,13 +18882,13 @@
         "level": "",
         "location": "New Jersey",
         "name": "New Jersey Special Statutory Court",
-        "notes": "Need more information about this court.",
+        "notes": "Need more information about this court. GPT4/Google do not recognize it.",
         "regex": [
             "New Jersey Special Statutory Court",
             "New Jersey Special Statutory Tribunal"
         ],
         "system": "state",
-        "type": ""
+        "type": null
```

Comment:
Unfortunately, we are often basing the courts on actual courts we see in books.  but that there aren't long historical records for.